src/tests/FormCreateChecking.test.js

```diff
@@ -0,0 +1,89 @@
+import '@testing-library/jest-dom/extend-expect';
+import { render } from "@testing-library/react";
+import { screen, waitFor, within } from '@testing-library/dom';
+import {BrowserRouter as Router} from 'react-router-dom';
+import user from '@testing-library/user-event'
+
+import FormCreateChecking from '../components/FormCreateChecking'
+
+describe('FormCreateChecking test', () => {
+    const onSubmit = jest.fn();
+
+    beforeEach(() =>{
+        onSubmit.mockClear();
+        render(
+            <Router>
+                <FormCreateChecking/>
+            </Router>
+        );
+    });
+
+    it('Render checking form', () => {
+        const component = render(
+            <Router>
+                <FormCreateChecking/>
+            </Router>
+        );
```

Comment:
Read this article to fix the eslint issue: https://github.com/testing-library/eslint-plugin-testing-library/blob/main/docs/rules/render-result-naming-convention.md